packages/@aws-cdk/cloudformation-include/test/test-templates/nested/parent-export-stack.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
Shouldn't this be called `parent-two-children.json`? I don't know what the `import` / `export` part of a lot of these file names mean.

Reply:
The import/export indicates whether a stack is exporting a value to be imported by another stack, or importing a value that was exported by another stack. This doesn't need to be in the file name though.

Reply:
But where is that in the template? This is the contents of `parent-export-stack.json`:

```json

{
  "Resources": {
    "ChildStack": {
      "Type": "AWS::CloudFormation::Stack",
      "Properties": {
        "TemplateURL": "https://cfn-templates-set.s3.amazonaws.com/child-import-stack.json",
        "Parameters": {
          "MyBucketParameter": "some-magic-bucket-name"
        }
      }
    },
    "AnotherChildStack": {
      "Type": "AWS::CloudFormation::Stack",
      "Properties": {
        "TemplateURL": "https://cfn-templates-set.s3.amazonaws.com/child-import-stack.json",
        "Parameters": {
          "MyBucketParameter": "another-magic-bucket-name"
        }
      }
    }
  }
}
```

Where is the import / export here?

Reply:
it's not in the file, but the value gets passed from the parent. So the parameter comes from the parent, and since the child imports it, I figured export would work.

Reply:
I feel like that's not what I expect from the words "import" / "export" in CloudFormation. Those words to me mean using `Output` with `Export` , and then `Fn::ImportValue`. So a template using that name, but containing none of those elements, is surprising to me.